torchmultimodal/modules/layers/transformer.py

```diff
@@ -83,9 +81,7 @@ def forward(
             self.attn_dropout,
         )
 
-        context_layer = context_layer.permute(0, 2, 1, 3).contiguous()
```

Comment:
non blocking Q: This `contiguous` call is replaced by `make_contiguous` option in `shift_dim`. But in general calling `contiguous` after `permute` may not be necessary:

https://discuss.pytorch.org/t/call-contiguous-after-every-permute-call/13190/3

I guess we can still default `make_contiguous` as `True` just to be safe.

Reply:
I kept it because I was previously using `view` instead of `flatten` and `unflatten` right after `shift_dim` and I ran into errors if I didn't make contiguous. It may not be necessary anymore but I would lean towards keeping to be safe, unless we find a clear drawback in doing so.